meson.build

```diff
@@ -463,6 +463,41 @@ endif
 
 
 
+###############################################################
+# Library: Tcl (for pltcl)
+###############################################################
+
+tcl_dep = dependency('', required : false)
+tclopt = get_option('pltcl')
+if not tclopt.disabled()
+  possible_tcl_packages = ['tcl', 'tcl84', 'tcl8.4', 'tcl85', 'tcl8.5', 'tcl86', 'tcl8.6']
```

Comment:
I think this order is the wrong way - shouldn't it be tcl, 86, 85, 84, i.e. the newer versions first?

Perhaps we should just require a parameter to specify the version if 'tcl' isn't sufficient, instead of having a list of versions that'll just become outdated?

Reply:
The existing configure test uses tclConfig.sh - have you tried to figure out why / whether we ought to copy that?